src/WebApp/wwwroot/lib/bootstrap/package.json

```diff
@@ -91,7 +91,9 @@
     "url": "https://github.com/twbs/bootstrap/issues"
   },
   "license": "MIT",
-  "dependencies": {},
+  "dependencies": {
+    "bootstrap": "file:"
```

Comment:
?

Reply:
Dit is iets wat mijn omgeving automatisch terugstuurt ik zal het wel even verwijderen en nog een keer pushen.